src/test/java/hw2/ex2/SecondScenarioTest.java

```diff
@@ -0,0 +1,278 @@
+        driver.findElement(By.xpath("//li[@class='dropdown']")).click();
+        WebElement headerDropDownList = driver.findElement(By.
+                xpath("//ul[@class='dropdown-menu']"));
+        List<String> actualServiceHeaderTexts = Arrays.
+                asList(headerDropDownList.getText().split("\n"));
+        ArrayList<String> expectedServiceHeaderTexts = new ArrayList<>();
+        expectedServiceHeaderTexts.add("SUPPORT");
+        expectedServiceHeaderTexts.add("DATES");
+        expectedServiceHeaderTexts.add("SEARCH");
+        expectedServiceHeaderTexts.add("COMPLEX TABLE");
+        expectedServiceHeaderTexts.add("SIMPLE TABLE");
+        expectedServiceHeaderTexts.add("USER TABLE");
+        expectedServiceHeaderTexts.add("TABLE WITH PAGES");
+        expectedServiceHeaderTexts.add("DIFFERENT ELEMENTS");
+        expectedServiceHeaderTexts.add("PERFORMANCE");
+        assertEquals(actualServiceHeaderTexts, expectedServiceHeaderTexts);
+
+        // 6. Click on Service subcategory in
+        // the left section and check that drop down contains options
+        // There is a clicking on "Service" in the left section
+        // and getting actual texts from service-dropdown menu. Then
+        // we create a list of expected text and compare them
+        // NB! Soft Assert can be placed here 'cause we can
+        // continue test even if there elements aren't exist
+        // (we'll not use them in the further steps)
+        driver.findElement(By.className("menu-title")).click();
+        WebElement leftSectionDropDownList = driver.findElement(By.
+                xpath("//ul[@class='sub']"));
+        List<String> actualServiceLeftSectionTexts = Arrays.
+                asList(leftSectionDropDownList.getText().split("\n"));
+        ArrayList<String> expectedServiceLeftSectionTexts = new ArrayList<>();
+        expectedServiceLeftSectionTexts.add("Support");
+        expectedServiceLeftSectionTexts.add("Dates");
+        expectedServiceLeftSectionTexts.add("Complex Table");
+        expectedServiceLeftSectionTexts.add("Simple Table");
+        expectedServiceLeftSectionTexts.add("Search");
+        expectedServiceLeftSectionTexts.add("User Table");
+        expectedServiceLeftSectionTexts.add("Table with pages");
+        expectedServiceLeftSectionTexts.add("Different elements");
+        expectedServiceLeftSectionTexts.add("Performance");
+        sa.assertEquals(actualServiceLeftSectionTexts,
+                expectedServiceLeftSectionTexts);
+
+        // 7. Open through the header menu Service ->
+        // Different Elements Page
+        // There is a simple opening of new
+        // page -- https://epam.github.io/JDI/different-elements.html
+        // and checking if the page is really opened
+        driver.findElement(By.xpath("//li[@class='dropdown']")).click();
+        driver.findElement(By.linkText("Different elements")).click();
+        assertEquals(driver.getCurrentUrl(),
```

Comment:
Why do you use `Assert.assertEquals` here?

Reply:
I should compare the actual URL (I got it with *driver.getCurrentUrl()*) with expected URL (I wrote it as the second parameter in the assert-method). `Assert.assertEquals` can complete it and inform me if something goes wrong. That's why I use this method here